src/main/java/seedu/address/logic/commands/EditEventCommand.java

```diff
@@ -109,6 +122,11 @@ public String toString() {
      */
     public static class EditEventDescriptor {
         private EventName name;
+        private SportString sport;
+
+        private Venue venue;
+
+        private Set<Person> participants;
```

Comment:
Great job on implementing our change with Events keeping track of which participants they hold rather than participants keeping track of which event they are taking part in.